trafficlight/model.py

```diff
@@ -1,14 +1,26 @@
 from pydantic import BaseModel, Field
+from .config import config
 
 
 class ProtoModel(BaseModel):
-    method: int = Field(alias="type")
-    request: str | None
-    response: str | None = Field(alias="payload")
+    if config.use_golbat_raws:
```

Comment:
I just checked the changes, I may not understand them so pls correct me if needed.

You introduced a new config variable "use_golbat_raws" -> it should support golbat raw forwarding to Trafficlight, right?

The default of this config variable is false, so it doesn't change behavior, we asked for that right?

But exactly that thing you changed in here, instead of using old definitions you use now the new definitions without Field aliases?!

I don't understand that part of python, but wouldn't it be way smarter to use pydantic field in the right way?

I will try to cover that in a [PR](https://github.com/ccev/TrafficLight/pull/17)